readers/mongodb/messages_test.go

```diff
@@ -6,47 +6,42 @@ package mongodb_test
 import (
 	"context"
 	"fmt"
-	"os"
 	"testing"
 	"time"
 
 	writer "github.com/mainflux/mainflux/consumers/writers/mongodb"
 	"github.com/mainflux/mainflux/pkg/transformers/senml"
+	"github.com/mainflux/mainflux/pkg/uuid"
 	"github.com/mainflux/mainflux/readers"
 	reader "github.com/mainflux/mainflux/readers/mongodb"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	log "github.com/mainflux/mainflux/logger"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
 const (
 	testDB      = "test"
 	collection  = "messages"
-	chanID      = "1"
 	subtopic    = "subtopic"
-	msgsNum     = 42
+	msgsNum     = 100
```

Comment:
Maybe it's better to say `numMsgs` - number of messages (just an aesthetic remark).